microcosm_pubsub/tracing.py

```diff
@@ -0,0 +1,96 @@
+"""
+Support optional tracing using Dynatrace oneagent.
+
+If the dynatrace SDK is not installed, does nothing.
+
+"""
```

Comment:
Should this be `tracing.py` or `dynatracing.py`

Reply:
I went with tracing.py as we could perhaps choose between different tracing backends here - though the API would definitely need some work.

Reply:
This can be the "opinionated" part. 👍